post_installation_configuration/cluster-tasks.adoc

```diff
@@ -94,3 +94,25 @@ Understand and configure Pod disruption budgets.
 
 include::modules/nodes-pods-pod-disruption-about.adoc[leveloffset=+2]
 include::modules/nodes-pods-pod-disruption-configuring.adoc[leveloffset=+2]
+
+[id="post-install-rotate-cloud-creds"]
+== Rotating cloud provider credentials
+Some organizations require rotating the cloud provider credentials that were used to install {product-title} after installation. The xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator (CCO)] manages cloud provider credentials and can be configured to operate in several different modes. The process for rotating cloud credentials depends on the mode that the CCO is configured to use.
+
+[id="cred-rotation-cco-mode-mint"]
+=== Rotating credentials with the CCO in mint mode
+When the CCO is configured to operate in _mint mode_, it uses the provided admin-level cloud credential to create new credentials for each component with only the specific permissions that are required. Rotating credentials in mint mode requires the manual removal of the component credentials that were created by the removed credential.
+
+. Update the admin credentials root secret for your cloud provider.
+
+. Look up every credential that has been minted for the `credentialsRequest` CR and delete it from the cloud provider.
+
+. Delete each component secret that is referenced by the individual `credentialsRequest` objects.
+
+The CCO uses the new admin-level cloud credential to create new credentials for each component.
+
+[id="cred-rotation-cco-mode-passthrough"]
+=== Rotating credentials with the CCO in passthrough mode
+When the CCO is configured to operate in _passthrough mode_, it passes the provided cloud credential to the components that request cloud credentials. The credential must have permissions to perform the installation and complete the operations that are required by components in the cluster, but does not need to be able to create new credentials.
+
+After rotating the passthrough mode cloud credential, it can take up to 11 hours to propagate the new credential to all of the components that need it. During this time, the cluster might not be fully operational. To reduce the time to propagate the new credential, delete the CCO pod. Deleting the CCO pod causes the CCO to immediately propagate the new credential, which can take up to an hour to complete.
```

Comment:
Do we know how to locate and delete that pod?

Reply:
For 4.7, if we rotated the admin credentials root secret, it doesn't need to wait for 11 hours, cco will immediately propagate the new credential to all related components.  The only step the user need to do is manually rotating the admin credentials root secret in kube-system namespace. and I saw @dgoodwin is going to backport it to 4.6 and 4.5.
The thing we need to care is that all related components respect the credentials update, but I haven't finished the joint tests,

Reply:
@lwan-wanglin Thanks - yes, I should have added a comment about this specifically. The current plan is:

1. Publish this PR to address the current state of things.
2. Create a new PR to merge into master and 4.7 without the stuff about waiting.
3. When the 4.6 and 4.5 backport go out, update the 4.5 and 4.6 docs to state "as of 4.5.z" there is no need to wait.